doc/source/fuzzycmeans.rst

```diff
@@ -0,0 +1,76 @@
+Fuzzy C-means
+=============
+
+`Fuzzy C-means <https://en.wikipedia.org/wiki/Fuzzy_clustering#Fuzzy_C-means_clustering>`_ is a method for clustering that provides membership values for some user-defined number of clusters instead of hard classifications (e.g., K-means).
```

Comment:
```suggestion
`Fuzzy C-means <https://en.wikipedia.org/wiki/Fuzzy_clustering#Fuzzy_C-means_clustering>`_ is a clustering method that provides cluster membership weights instead of "hard" classification (e.g. K-means).
```